docs/source/release_notes/release-notes.rst

```diff
@@ -69,6 +69,10 @@ develop
          - Messages to the `slow-lock-log` now log at `WARN` rather than `INFO`, these messages can indicate a problem so we should be sure they are visible.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2828>`__)
 
+    *    - |fixed|
+         - SweepResults.getCellTsPairsExamined now returns the correct result when sweep affects multiple batches.
```

Comment:
Let's explain the problem we fixed here.

"SweepResults.getCellTsPairsExamined now returns the correct result when sweep is run over multiple batches. Previously, the result would only count cell-ts pairs examined in the last batch."